lessons/projects/1.mdx

```diff
@@ -710,4 +722,4 @@ Can a view function modify the state of the blockchain?
 What can we use a smart contract for?
 
 **Now, go to the community in Discord to share your new Open Sourcerer
-powers**......and find the answers too!!! 
+powers**......and find the answers too!!!
```

Comment:
For future reference, check if you modified lines by accident 😊 
In this case it's really not important, but in big PRs, it creates a little more unnecessary work for the reviewer.